great_expectations/checkpoint/checkpoint.py

```diff
@@ -297,7 +298,10 @@ def run(
         )
 
         checkpoint_result = self._construct_result(run_id=run_id, run_results=run_results)
-        self._run_actions(checkpoint_result=checkpoint_result)
+
+        action_results = self._run_actions(checkpoint_result=checkpoint_result)
+        # TODO(cdkini): Figure out how to handle action results
+        print(action_results)  # We could add these to our checkpoint result or logger.warning?
```

Comment:
This is where I'm unsure but the goal with this whole effort is to make action results more transparent - we currently silently fail or skip

Reply:
If we log, there's still a chance can lose track of it.
I'd lean towards updating the checkpoint result but we need to be careful about what we add there - don't want it becoming as cumbersome as its V0 counterpart.